src/CachePlugin.php

```diff
@@ -34,31 +35,86 @@
     private $config;
 
     /**
+     * Cache directives indicating if a response can not be cached.
+     *
+     * @var array
+     */
+    private $noCacheFlags = ['no-cache', 'private'];
```

Comment:
AFAIK you can actually cache the result if response has `no-cache` directive. It just means you must always revalidate before actually using the cached result. If origin server returns `304 Not Modified` you can use the cached result.

That said the easy implementation is just not to cache. Only difference is some bandwith is wasted, but implementation is much easier.

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.1